jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/ExecutorServiceFactory.java

```diff
@@ -49,6 +49,9 @@
 public class ExecutorServiceFactory {
 
 	private final static String mode = System.getProperty( "org.jbpm.cdi.executor.mode", "singleton" );
+	
+	private final static String availableJobsExecutorName = System.getProperty(
+	    "org.jbpm.cdi.available.jobs.executor.name", "java:module/AvailableJobsExecutor" );
```

Comment:
Would it be possible to change this system property to something easier and self-explanatory? Since there is already `org.jbpm.cdi.executor.mode` used in this class, I would suggest using `org.jbpm.cdi.executor.jndi` for this new property.